sentinel-adapter/sentinel-apache-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/SentinelDubboConsumerFilter.java

```diff
@@ -98,7 +98,7 @@ private Result syncInvoke(Invoker<?> invoker, Invocation invocation) {
             throw e;
         } finally {
             if (methodEntry != null) {
-                methodEntry.exit();
+                methodEntry.exit(1, invocation.getArguments());
```

Comment:
Very careful code! How about also update `SentinelDubboConsumerFilter` in `sentinel-dubbo-adaptor` module to keep them in line, though they are for different version of dubbo.

Reply:
ConsumerFilter in `sentinel-dubbo-adaptor` don't take param when entry. Should I add param when entry and exit?

Reply:
Yes, it does't take param when entry currently, users may feel confused and suggest them to be consistent.